src/schema/artwork/index.ts

```diff
@@ -780,7 +786,7 @@ export const ArtworkType = new GraphQLObjectType({
       cached,
     }
   },
-})
+} as any)
```

Comment:
These are rabbit holes that turn into black holes very quickly, so I as any'd it